readme.scroll

```diff
@@ -18,7 +18,7 @@ startColumns 4
 
 * So your data, your query language, your schemas, your display language are all in the same simple plain-text notation: Tree Notation. Of course, there's also a lot of Javascript glue.
 
-? Why is there so little documentation?
+? Is there a Rationale for current release schedule/date ?
```

Comment:
I agree the answer does not match the question.

How about changing the question to: `? Since this is so simple, what hasn't it been done before?`